### PR TITLE
Egress Kubernetes Services Automated test

### PR DESCRIPTION
**Pull request by nschhina**
Egress Kubernetes Services Test

### CODE REVIEW

**Comment by nschhina** [comment]
This k8s test failure is not associated with my test

**Comment by frankbu** [comment]
Missing cleanup instruction in the doc (delete my-httpbin) is causing the next test to fail.

File: content/en/docs/tasks/traffic-management/egress/egress-kubernetes-services/test.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+_verify_contains snip_kubernetes_externalname_service_to_access_an_external_service_3 "\"Host\": \"my-httpbin.default.svc.cluster.local"
+
+# apply dr
+snip_kubernetes_externalname_service_to_access_an_external_service_4
+_wait_for_istio destinationrule default my-httpbin
+
+_verify_contains snip_kubernetes_externalname_service_to_access_an_external_service_5 "\"X-Envoy-Decorator-Operation\": \"my-httpbin.default.svc.cluster.local:80/*\""
+
+# cleanup istio
+snip_cleanup_of_kubernetes_externalname_service_1
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/egress/egress-kubernetes-services/test.sh:51]: Move this after line 68

**Pull request merged**